integration_test/tac_test.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:concordium_wallet/screens/terms_and_conditions/widget.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:concordium_wallet/main.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  // TODO Remove / change this test when we remove temp reset T&C page
+  testWidgets('Confirming T&C on the initial page brings us to the reset T&C page and resetting accepted T&C brings us back.', (tester) async {
+    // Load app widget.
+    await tester.pumpWidget(const App());
```

Comment:
```suggestion
    start.main();
```

You don’t need to pump it twice. Just start the app from main entrypoint.

Use import statement is
```
import 'package:concordium_wallet/main.dart' as start;
```